src/components/app-layout/AppLayout.js

```diff
@@ -23,6 +23,10 @@ const Noop = () => null
     flexShrink: 0,
     flexDirection: 'column',
   },
+  fileUpload: {
```

Comment:
I wonder why this happening on your machine. Doesn't happen to nor did we have bug repots, hmmmm

Reply:
Wow, after switching to yarn now I have it too 💥